dk.sdu.se4.commonenemy/osgi.bnd

```diff
@@ -0,0 +1,4 @@
+#-----------------------------------------------------------------
+# Use this file to add customized Bnd instructions for the bundle
+#-----------------------------------------------------------------
+Export-Package: dk.sdu.se4.commenenemy
```

Comment:
not needed

Reply:
https://bnd.bndtools.org/heads/export_package.html

vi kan enten expotere dem ved maven prom eller med bnd :) 

men når vi skal lave FX CommenEnemy er det en god pratice at bruge Export packet :)  da mange forskellig moduler skal bruge dem 
men bung og grund et det et spørgesmål om vi skal benytte bundle eller maven packet export

Reply:
Jeg fjerne dem :)